js/app.js

```diff
@@ -0,0 +1,30 @@
+var toDoApp = angular.module('toDoApp', []);
+function ToDoController($scope) {
+  $scope.tasks = [
+    { todo: 'Cleaning', completed: false }, 
+    { todo: 'Shopping', completed: false }, 
+    { todo: 'Laundry', completed: false },
+    { todo: 'Ironing', completed: false }
+  ];
+    
+  $scope.getToDoCount = function () {
+    return $scope.tasks.length;
+  };
+    
+    
+  $scope.addToDo = function () {
+    $scope.tasks.push({todo:$scope.formToDoText, completed:false});
+    $scope.formToDoText = '';
+  };
+    
+  $scope.clearAll = function () {
+      $scope.tasks = _.filter($scope.tasks, function(todo){
```

Comment:
`_` is usually if you're using a library called underscore or lo-dash - but, JS has filter built in!